python_modules/dagster/dagster/_daemon/sensor.py

```diff
@@ -82,6 +82,16 @@
 FINISHED_TICK_STATES = [TickStatus.SKIPPED, TickStatus.SUCCESS, TickStatus.FAILURE]
 
 
+# sensor, elapsed, min_interval
+ElapsedInstrumentation = Callable[[Optional[RemoteSensor], Optional[float], int], None]
+
+
+def default_elapsed_instrumentation(
+    sensor: Optional[RemoteSensor], elapsed: Optional[float], min_interval: int
```

Comment:
i think there will always be a RemoteSensor in this daemon and this does not need to be Optional